_pages/tools/slack/user-management.md

```diff
@@ -88,11 +88,6 @@ ensure they're in line with our values of diversity and inclusion. Inclusion Bot
 Ideally, submitters will check before requesting, but it's the approver's
 responsibility to verify.
 
-**Emoji of TTS members' faces, family, pets, etc**: When reviewing, emoji approvers
-**MUST** ask for clear, affirmative consent from the person whose image is being used.
-Ideally, submitters will check before requesting, but it's the approver's
-responsibility to verify.
-
```

Comment:
I'm curious if you meant to remove this, or if it slipped in accidentally with a merge somewhere. It's a pretty new policy, so if we're reverting so quickly, I'd be interested in understanding why.